src/main/java/com/streamr/client/options/StreamrClientOptions.java

```diff
@@ -12,7 +12,7 @@
     private SigningOptions signingOptions = SigningOptions.getDefault();
     private EncryptionOptions encryptionOptions = EncryptionOptions.getDefault();
     private boolean publishSignedMsgs = false;
-    private String websocketApiUrl = "wss://www.streamr.com/api/v1/ws?controlLayerVersion=1&messageLayerVersion=30";
+    private String websocketApiUrl = "wss://www.streamr.com/api/v1/ws?controlLayerVersion=2&messageLayerVersion=32";
```

Comment:
Can we use the `LATEST_VERSION` constants here too, to avoid having to remember to update this every time?